src/control/content/content.js

```diff
@@ -378,19 +381,24 @@ class Content extends React.Component {
    */
   onMultipleLocationSubmit(locations) {
     locations = locations.filter((location) => typeof location === "object");
+    console.log(locations)
```

Comment:
lets remove the console log and we should be good to go

Reply:
removed